asab/zookeeper/__init__.py

```diff
@@ -1,7 +1,6 @@
 import logging
 
 from .service import ZooKeeperService
-from .container import ZooKeeperContainer
```

Comment:
Well, you include random irrelevant stuff in merge request ... create a small one for these next time. please.

Reply:
I was trying to make PEP8 happy, to make branch mergable. 
I don't know, why there were these relikts in master. I guess someone have merged something, that didn't pass the unit tests..